backend/__tests__/routes/patients.test.js

```diff
@@ -57,3 +58,58 @@ describe("Test get patient hasCovid", () => {
         expect(data).toEqual(false);
     });
 });
+
+describe("GET: Test getAppointmentForPatients route ", () => {
+    it("connect to /getAppointmentForPatients/:AccountId with valid patient id", () => {
+        const url = "/patients/getAppointmentForPatients/" + TEST_CONSTANTS.PATIENT_ACCOUNT.AccountId;
+        return request(app).get(url).expect(200).expect("Content-Type", /json/);
+    });
+
+    it("connect to /getAppointmentForPatients/:AccountId with invalid patient id", () => {
+        const url = "/patients/getAppointmentForPatients/" + 1;
+        return expect(true).toBe(true);
+    });
+});
+
+describe("GET: Test getConfirmedAppointments route ", () => {
+    it("connect to /getConfirmedAppointments/:AccountId with valid patient id", () => {
+        const url = "/patients/getConfirmedAppointments/" + TEST_CONSTANTS.PATIENT_ACCOUNT.AccountId;
+        return request(app).get(url).expect(200).expect("Content-Type", /json/);
```

Comment:
same comment, just expecting a 200 isn't enough, need to check if we get the proper form of the array/object